zerver/tests/test_openapi.py

```diff
@@ -929,12 +930,29 @@ def test_attributes(self) -> None:
             if path in EXCLUDE:
                 continue
             for method, operation in path_item.items():
-                # Check if every file has an operationId
+                # Check if every file has an operationId.
                 assert "operationId" in operation
                 assert "tags" in operation
                 tag = operation["tags"][0]
                 assert tag in VALID_TAGS
                 for status_code, response in operation["responses"].items():
+                    # Validate the documented examples for each event type
+                    # in api/get-events for the documented event schemas.
+                    if path == "/events" and method == "get":
+                        properties = response["content"]["application/json"]["schema"]["properties"]
+                        # Only check the success response with the "events" array.
+                        if "events" in properties:
+                            for _, event_type in enumerate(properties["events"]["items"]["oneOf"]):
+                                event_array = [event_type["example"]]
+                                content = {
+                                    "queue_id": "fb67bf8a-c031-47cc-84cf-ed80accacda8",
+                                    "events": event_array,
+                                    "msg": "",
+                                    "result": "success",
+                                }
+                                assert validate_against_openapi_schema(
+                                    content, path, method, status_code
+                                )
                     schema = response["content"]["application/json"]["schema"]
```

Comment:
nit: can simplify the new code slightly by putting it after this line, and using this local

Reply:
Done.